apps/charterafrica/src/lib/ecosystem/ecosystem.js

```diff
@@ -0,0 +1,73 @@
+import {
+  getListFromAirtable,
+  processToolFromAirtable,
+  processOrganisationFromAirTable,
+  processContributorFromAirtable,
+} from "@/charterafrica/lib/ecosystem/airtable";
+import {
+  createCollection,
+  ORGANIZATION_COLLECTION,
+  CONTRIBUTORS_COLLECTION,
+  TOOL_COLLECTION,
+  ECOSYSTEM_CONFIG,
+} from "@/charterafrica/lib/ecosystem/models";
+import api from "@/charterafrica/lib/payload";
+
+export const updateEcosystemList = async (req, res) => {
+  const config = await api.findGlobal(ECOSYSTEM_CONFIG, {});
+  const args = {
+    baseId: config.airtableBase,
+    tableIdOrName: config.toolsTableName,
+  };
+  const response = await getListFromAirtable(args);
```

Comment:
Are we returning the (whole) list or are we supposed to create collection items from this list?

Reply:
We are supposed to create/update a collection @kilemensi.
This part will be handled by PR #444

Reply:
🤔 ... I think creating collections should be done here @koechkevin . i.e. given the list of this contributors/organisations/tools, create the respective items in the CMS.

#444 is supposed to add flesh to these items but these items need to already exist in the CMS.